faq.md

```diff
@@ -8,18 +8,22 @@ permalink: /faq/
 
 ## Rebble
 
+### Where is the FAQ?
+
+Well, right here! At least, for frequently asked questions about *The Rebble Alliance*. For frequently asked questions about how to use the Rebble Web Services with your Pebble smartwatch, head to [the setup and troubleshooting how-to](/howto) and scroll down.
+
 ### What is Rebble?
 
 Rebble began as a supplement to the [official Pebble documentation](https://developer.pebble.com/).  After [Pebble ceased operations](https://blog.getpebble.com/2016/12/07/fitbit/), it morphed into the unofficial spearhead organization for continuing the advancement of the Pebble platform via the amazing Pebble developer community.  
 
 ### What does Rebble do?
 
-Our present focus is on replacing key pieces of the Pebble ecosystem: the appstore, firmware, and mobile apps.  At a higher level, our goal is to maintain and advance Pebble functionality, in the absence of *Pebble Technology Corp*.  
+Our present focus is on replacing key pieces of the Pebble ecosystem: the appstore and related web services, and later on down the road, potentially even firmware and mobile apps.  At a higher level, our goal is to maintain and advance Pebble functionality, in the absence of *Pebble Technology Corp*.  
```

Comment:
Not super clear on the motivation behind this tweak? 🔎

Reply:
The language change is meant to make the app store and other web services distinct from RebbleOS and any potential mobile apps projects. One is already being used by thousands of people, the others are much further ahead. Also, a lot of people seem to come to the Discord server or post in the subreddit thinking Rebble is all about firmware, but that hasn't been the case in a long time and might not be the case for a long time either.

Reply:
I can change it back if anyone disagrees.